### PR TITLE
Update form_types.rst

### PR DESCRIPTION
**Pull request by artggd**
The syntax `twig.form.ressources` has been deprecated since v2.6 and does not work anymore.

This PR comply with new twig configuration on form_theme https://symfony.com/doc/current/reference/configuration/twig.html

### CODE REVIEW

**Comment by greg0ire** [comment]
Please fill the PR template (you're targetting master without explaining why).

**Comment by artggd** [comment]
I opened this PR on the wrong branch should've been on the 3.x, I may re-open it when I find some time to do so.

**Comment by OskarStark** [comment]
we can switch the branches

File: Resources/doc/reference/form_types.rst [hunk 1]
```diff
@@ -387,9 +387,8 @@ In order to use them, you'll need to perform a bit of setup:
         # app/config/config.yml
 
         twig:
-            form:
-                resources:
-                    - 'SonataCoreBundle:Form:datepicker.html.twig'
+            form_themes
```
Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: `:` missing

**Comment by greg0ire** [comment]
> we can switch the branches

it was done, this is why there are 50000 commits

**Comment by OskarStark** [comment]
i saw it and removed my comment :)

**Pull request closed without merging**